frontend/cypress/integration/login_page_spec.js

```diff
@@ -4,9 +4,6 @@ describe("The Login Page", function() {
     cy.get('input[name="email"]').type("Nonexistent email");
     cy.get('input[name="password"]').type("password");
     cy.get('button[type="submit"]').click();
-
-    cy.wait(3000);
-
-    cy.url().should("include", "/login");
+    cy.url().should("not.include", "/dashboard");
```

Comment:
The problem with this approach is that it doesn't take into account the fact that the login is not instant and that in any case some time will have to pass between the clicking of the login button and the change of the url.
If you were to change it to the opposite test case, that is   `cy.url().should("not.include", "/login");` the test would still pass :D

Reply:
![image](https://user-images.githubusercontent.com/17446975/57151543-e2170100-6dd9-11e9-9d86-11cef2b027b6.png)
Aš galiu parefreshint page ir tada padaryti checką, jaučiu, geriausias sprendimas bus.